.pre-commit-config.yaml

```diff
@@ -82,7 +82,6 @@ repos:
     # files: "^notebooks/(api|tutorials|admin)"
     hooks:
       - id: nbqa-isort
-      - id: nbqa-black
```

Comment:
is the ruff formatter working as expected on notebooks without this? Like cell re-indents and max line width 88 (or whatever the default is for ruff)

Reply:
Yup alr tested on my machine by deliberately adding malformatted code.